packages/core/src/types.ts

```diff
@@ -86,7 +86,9 @@ export const aggregateByList = [
 export type AggregateBy = typeof aggregateByList[number];
 
 export const sortByList = ['Degree', 'Size', 'Deviation'] as const;
+export const sortByOrderList = ['Ascending', 'Descending'] as const;
 export type SortBy = typeof sortByList[number];
+export type SortByOrder = typeof sortByOrderList[number];
```

Comment:
Could this be simplified?
```suggestion
export type SortBy = typeof sortByList[number];
export type SortByOrder = 'Ascending' | 'Descending';
```